Update set-up-anti-phishing-policies.md (#4145)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md

>   > [!IMPORTANT]

Reviewer: ```suggestion

    > [!IMPORTANT]
```

  >

>   > **Recommendation for enabling safety tip that will appear based on first time contact between sender and recipient(s)**:

Reviewer: ```suggestion
  > **Recommendation for enabling a safety tip that will appear during the first-time contact between the sender and recipient(s)**:
```
